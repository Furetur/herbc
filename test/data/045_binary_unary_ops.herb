fn main() {
    print(1 + 2 * 3);
    print(" ");
    print(2 * 3 * -1);
    print(" ");
    print(10 + 3 / 3 + (10 * 10) / 9);
    print(" ");
    print(100 % 9 + 1);
    print(" ");
    print((1 < 1) || (6 % 2 == 0));
    print(" ");
    print(true || true);
    print(false || true);
    print(false || false);
    print(true || false);
    print(" ");
    print(true && true);
    print(true && false);
    print(false && true);
    print(false && false);
    print(!true);
    print(!false);
    print(" ");
    print(1 <= 2);
    print(1 <= 1);
    print(2 <= 3);
    print(2 <= 2);
    print(4 == 4);
    print(4 != 3);
    print(4 > 3);
    print(4 >= 4);
}
