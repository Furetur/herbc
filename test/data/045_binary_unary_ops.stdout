7 -6 22 2 true truetruefalsetrue truefalsefalsefalsefalsetrue truetruetruetruetruetruetruetrue